### PR TITLE
Create cloudcma.com.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

File: src/chrome/content/rules/cloudcma.com.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		www.ztc.cloudcma.com
+-->
+<ruleset name="cloudcma.com">
+	<target host="cloudcma.com" />
+	<target host="www.cloudcma.com" />
+	<target host="agent.cloudcma.com" />
+	<target host="blog.cloudcma.com" />
+	<target host="partner.cloudcma.com" />
+	<target host="powerpack.cloudcma.com" />
+	<target host="retech.cloudcma.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/cloudcma.com.xml]: `reports3.cloudcma.com`

File: src/chrome/content/rules/cloudcma.com.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!--
+	Invalid certificate:
+		www.agent.cloudcma.com
+		www.partner.cloudcma.com
+		www.powerpack.cloudcma.com
+		www.retech.cloudcma.com
+		www.tomferry.cloudcma.com
+		webinar.cloudcma.com
+		www.webinar.cloudcma.com
+		www.ztc.cloudcma.com
```
Comment (J0WI) [inline src/chrome/content/rules/cloudcma.com.xml]: Wildcard DNS record

Comment (TotalCaesar659) [reply src/chrome/content/rules/cloudcma.com.xml]: What do I need to do? Remove them?

Comment (J0WI) [reply src/chrome/content/rules/cloudcma.com.xml]: Please add a comment similar to https://github.com/EFForg/https-everywhere/search?q=wildcard+dns

**Pull request merged**